resources-domain/lambdas/sched-importer/src/khpMappings.ts

```diff
@@ -558,9 +563,16 @@ export const KHP_MAPPING_NODE: MappingNode = {
           info: ({ currentValue }) => currentValue,
           value: ({ currentValue }) => {
             const { postalCode, city, region, province, country } = currentValue;
-            return [postalCode, city, region, province, country]
+            const text = [postalCode, city, region, province, country]
               .filter(Boolean)
               .join(', ');
+            const filterPath = [
```

Comment:
This could maybe do with an explanatory comment. Someone without full context in the future would have to puzzle through why this returns this kind of string.